lms/static/sass/discussion/views/_create-edit-post.scss

```diff
@@ -115,8 +117,8 @@
   }
 
   input[type=text].field-input {
-    @include box-sizing(border-box);
-    border: 1px solid $gray-l2;
+    box-sizing: border-box;
+    border: 1px solid $forum-color-border;
     border-radius: 3px;
     padding: 0 $baseline/2;
     height: 40px;
```

Comment:
Anyway to convert this pixel value to something relative like `em`? If a user manually increases the font-size, and if they need something that's larger than 40px, could look funky.